packages/components/popper/src/arrow.ts

```diff
@@ -1,12 +1,14 @@
 import { buildProps } from '@element-plus/utils'
 
 import type { ExtractPropTypes } from 'vue'
+import type Arrow from './arrow.vue'
 
-export const usePopperArrowProps = buildProps({
+export const popperArrowProps = buildProps({
   arrowOffset: {
     type: Number,
     default: 5,
   },
 } as const)
+export type PopperArrowProps = ExtractPropTypes<typeof popperArrowProps>
 
-export type UsePopperArrowProps = ExtractPropTypes<typeof usePopperArrowProps>
+export type PopperArrowInstance = InstanceType<typeof Arrow>
```

Comment:
It looks like you modified the export name of the types, I suggest add deprecated to the previous name.

// reference

```ts
export type PopperArrowInstance = InstanceType<typeof Arrow>

/** @deprecated use `PopperArrowInstance` instead, and it will be deprecated in the next major version */
export type ElPopperArrowInstance = PopperArrowInstance
```